requirements.txt

```diff
@@ -0,0 +1,21 @@
+annotated-types==0.6.0
```

Comment:
Please consider splitting this file to dev and prod requirements. It's not must have but best practice